.travis.yml

```diff
@@ -14,7 +14,9 @@ before_script:
   - travis_retry composer self-update
   - travis_retry composer update --no-interaction --prefer-source
 
-script: make test-coverage-clover
+script:
```

Comment:
See https://github.com/graze/formatter/blob/master/Makefile#L23-L27.

You could go a step further, write a custom matrix with an env variable for the php image to use and make use of it in Makefile, with a default.